components/ProductsPage/components.tsx

```diff
@@ -0,0 +1,282 @@
+import styled from "@emotion/styled";
+import { keyframes } from "@emotion/react";
+import { constants } from "../../utils";
+import { OOLogoIcon } from "../Icons";
+import { PrimaryLink } from "../Link";
+
+const { QUERIES, COLORS } = constants;
+const PulseRingAnimation1 = keyframes`
+  0% {
+    transform: scale(0.5, 0.5);
+    opacity: 0.0;
+  }
+
+  50% {
+    opacity: 0.1;
+  }
+
+  100% {
+    transform: scale(1.05, 1.05);
+    opacity: 0.0;
+  }
+`
+
+const PulseRingAnimation2 = keyframes`
+  0% {
+    transform: scale(0.5, 0.5);
+    opacity: 0.0;
+  }
+
+  50% {
+    opacity: 0.15;
+  }
+
+  100% {
+    transform: scale(1.05, 1.05);
+    opacity: 0.0;
+  }
+`
+
+const PulseRingAnimation3 = keyframes`
+  0% {
+    transform: scale(0.5, 0.5);
+    opacity: 0.0;
+  }
+
+  50% {
+    opacity: 0.3;
+  }
+
+  100% {
+    transform: scale(1.05, 1.05);
+    opacity: 0.0;
+  }
+`
+
+export const Container = styled.section`
+
+`;
+
+export const Content = styled.div`
+  max-width: 1200px;
+  margin: auto;
+`;
+
+export const CoverContainer = styled(Container)`
+  background-color: var(--gray-300);
+`;
+
+export const CoverContent = styled(Content)`
+  padding: 30px 15px 0;
+
+  @media ${QUERIES.tabletAndUp} {
+    padding: 50px 20px 0;
+  }
+
+  @media ${QUERIES.laptopAndUp} {
+    padding: 50px 40px 0;
+  }
+`;
+
+export const OOContent = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+export const CoverText = styled.div`
+  margin: 0 0 20px;
+  width: 100%;
+
+  @media ${QUERIES.tabletAndUp} {
+    margin: 0 0 40px;
+  }
+
+  @media ${QUERIES.laptopAndUp} {
+    width: 55%;
+  }
+`;
+
+export const DesktopLogoContainer = styled.div`
+  max-width: 400px;
+  width: 40%;
+  display: none;
+
+  @media ${QUERIES.laptopAndUp} {
+    display: block;
+  }
+`;
+
+export const MobileLogoContainer = styled.div`
+  max-width: 280px;
+  width: 100%;
+  margin: 15px auto 0;
+  display: block;
+
+  @media ${QUERIES.tabletAndUp} {
+    max-width: 350px;
+  }
+
+  @media ${QUERIES.laptopAndUp} {
+    display: none;
+  }
+`;
+
+export const LogoHeadline = styled.div`
+  margin: 0 0 15px;
+  display: flex;
+  align-items: center;
+
+  span {
+    margin-left: 15px;
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 600;
+  }
+
+  svg {
+    width: 40px !important;
+    height: 40px !important;
+  }
+
+  @media ${QUERIES.tabletAndUp} {
+    margin: 0 0 25px;
+
+    svg {
+      width: 60px !important;
```

Comment:
Any way we could avoid !important here?

Reply:
Yes, `!important` was not even needed here. I deleted all `!important` from the project